Swift/Database.swift

```diff
@@ -127,7 +127,7 @@ public final class Database {
     ///   - document: The document.
     ///   - conflictHandler: The conflict handler closure which can be used to resolve it.
     /// - Returns: True if successful. False on failure
-    /// - Throws: An error on a failure.
+    /// - Throws: An error, if an exception happens.
```

Comment:
```
    /// - Returns: True if successful. False on failure
    /// - Throws: An error on a failure.
```

From the comments, feeling like, when there is a failure of save, it will throw an exception. And when there is any exception other than conflict, it will throw an exception. But if it is a conflict, and it will return false without an error.

Reply:
Just saw your comment. The comment seems wrong. The comment should match or similar to saveDocument with concurrencyControl. Maybe something like below.

```
Saves a document to the database. When write operations are executed concurrently and if conflicts occur, 
the conflict handler will be called. Use the conflict handler to directly edit the document to resolve the conflict. When the conflict handler returns 'true', the save method will save the edited document as the resolved document. If the conflict handler returns 'false', the save operation will be canceled with 'false' value returned as the conflict wasn't resolved. 
...
/// - Returns: True if successful. False if there is a conflict, but the conflict wasn't resolved as the conflict handler returns 'false' value.
/// - Throws: An error on a failure.
```